src/Http/Plugin/ExceptionPlugin.php

```diff
@@ -30,7 +30,7 @@ private function transformResponseToException(RequestInterface $request, Respons
             throw AuthorizationException::fromRequestAndResponse($request, $response);
         }
 
-        if ($responseCode >= 401 && $responseCode < 600) {
+        if ($responseCode >= 400 && $responseCode < 600) {
```

Comment:
Any reason why not use the `StatusCodeInterface:: STATUS_BAD_REQUEST`?

Reply:
IMHO it will actually lower readability.

The condition is now explicitly read "for all response codes between 400 and 600 do this".
Using the constant, it will look like:
"for all response codes between BAD_REQUEST and 600 do this". But this is not what the condition is intended for.

The constants IMO makes sense when comparing with specific response code, not with range of codes...